app/tree/gui/src/ts/twxplore/gui/geo/components/sidebar/Sidebar.tsx

```diff
@@ -5,7 +5,6 @@ import Sidebar from "react-sidebar";
 import {Select} from "./Select"
 import { ActiveSidebarItem } from './ActiveSidebarItem'
 import { Button } from 'reactstrap';
-const mql = window.matchMedia(`(min-width: 800px)`);
```

Comment:
Not sure what this is for, but there are React-ish ways to do auto-resizing.

Reply:
I don't use this sidebar component anymore. the mql line doesnt exist in the most recent commit idt (it was causing the ts tests in CI to fail)